build_scripts/build_runner.py

```diff
@@ -179,9 +179,23 @@ def run(self):
 
                 self.log.debug(completed_process.stdout)
             except subprocess.CalledProcessError as process_error:
-                self.log.error(process_error.stdout)
+                if self.stage == Stage.BUILD:
+                    self._parse_logs(process_error.stdout)
+                else:
+                    self.log.error(process_error.stdout)
                 raise
 
+    def _parse_logs(self, stdout):
+        self.log.debug(stdout)
+        output = ""
+
+        substring = ' error ' if platform.system() == 'Windows' else ' error:'
+        for string in stdout.split('\n'):
+            if substring in string:
```

Comment:
are you sure that you need all strings with ' error ' substring? Do you have an example of such errors?

Reply:
Examples:
mfx_av1_dec_decode.cpp(92): error C2220: warning treated as error - no 'executable' file generated
LINK : fatal error LNK1257: code generation failed